cli/src/ios/common.ts

```diff
@@ -1,21 +1,60 @@
 import { readFile, writeFile } from '@ionic/utils-fs';
+import { execSync } from 'child_process';
 import { resolve } from 'path';
 
 import c from '../colors';
 import { checkCapacitorPlatform } from '../common';
 import { getIncompatibleCordovaPlugins } from '../cordova';
 import { OS } from '../definitions';
 import type { Config } from '../definitions';
+import { logger } from '../log';
 import { PluginType, getPluginPlatform } from '../plugin';
 import type { Plugin } from '../plugin';
-import { isInstalled } from '../util/subprocess';
+import { isInstalled, runCommand } from '../util/subprocess';
 
 export async function checkIOSPackage(config: Config): Promise<string | null> {
   return checkCapacitorPlatform(config, 'ios');
 }
 
+function execBundler() {
+  try {
+    const bundleOutput = execSync('bundle &> /dev/null ; echo $?');
+    return parseInt(bundleOutput.toString());
+  } catch (e: any) {
+    return -1;
+  }
+}
+
+export async function checkBundler(config: Config): Promise<string | null> {
+  if (config.cli.os === OS.Mac) {
+    let bundlerResult = execBundler();
+    if (bundlerResult === 1) {
+      // Bundler version is outdated
+      logger.info(`Using ${c.strong('Gemfile')}: Bundler update needed...`);
+      await runCommand('gem', ['install', 'bundler']);
+      bundlerResult = execBundler();
+    }
+    if (bundlerResult === 0) {
+      // Bundler in use, all gems current
+      logger.info(`Using ${c.strong('Gemfile')}: RubyGems bundle installed`);
+    } else {
+      // Bundler not in use
+      logger.info(`No ${c.strong(
+        'Gemfile',
+      )} found, trying system-wide CocoaPods.
+Consider adopting Bundler, see: ${c.strong(
```

Comment:
I'm all for this, but I'll have to ask if we might want to remove the Consider adopting Bundler bit, since we don't have a Gemfile in the template.

Reply:
I'd say this is merely a recommendation. People are free to add their own `Gemfile`.

Reply:
I'm up for the changes to allow support for Bundler but I don't believe we should have this else statement. This is to avoid confusion for current users who don't use/won't need, Bundler.

Reply:
Yeah - I think that was the consensus. @saschpe can you pull that?